blog-admin-backend-org-page.el

```diff
@@ -38,6 +38,8 @@
 "
   "template for org-page's org post")
 
+(defvar config-file nil
```

Comment:
I think it would be better to have a prefix like `org-page-config-file` or `blog-admin-backend-hexo-config-file` (ahh although this one is too long😆...)